core/server/server.js

```diff
@@ -421,6 +422,11 @@ class Server {
       cert,
       key,
     }))
+    camp.handle(function (req, res, next) {
+      const frontendMaxAge = 300
```

Comment:
I guess we could pull this out to a setting if we wanted so that self-hosting users can tweak it. We've already got

```yml
cacheHeaders:
  defaultCacheLengthSeconds: 120
```

so we would probably need to split out into

```yml
cacheHeaders:
  defaultBadgeMaxAge: 120
  defaultFrontendMaxAge: 300
```

For the moment I've just left it hard-coded.

Reply:
Hard coded seems fine until someone asks.

Reply:
Agreed